Documentation/contributing/testing/unit.rst

```diff
@@ -123,3 +119,125 @@ run this in a terminal next to your editor:
     $ watchtest pkg/policy
 
 This shell script depends on the ``inotify-tools`` package on Linux.
+
+.. _migrate-gocheck:
+
+Migrating tests off of ``gopkg.in/check.v1``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Cilium codebase has historically used ``gopkg.in/check.v1`` to write unit and
+integration tests. Most of the functionality offered by that package is now
+present in `testing`_. What ``gocheck`` calls checkers are now available in
+``testify``, either in `assert`_ or `require`_ form.
+
+* ``check.Equals``: ``require.EqualValues``
+
+* ``check.HasLen``: ``require.Len``
+
+* ``check.ErrorMatches``: ``require.ErrorContains`` (doesn't support regex)
+
+* ``checker.Equals`` and ``checker.DeepEquals``: ``require.Equal``
+
+It is best to replace ``check.C.Assert`` with ``require`` calls instead of ``assert``, since the latter doesn't stop test execution if the assertion fails.
```

Comment:
s/since/because to clarify the causative relationship
```suggestion
It's best to replace ``check.C.Assert`` with ``require`` calls instead of ``assert`` because the latter doesn't stop test execution if the assertion fails.
```